src/main/java/com/rednavis/vaadin/showcase/jms/StartMessage.java

```diff
@@ -0,0 +1,16 @@
+package com.rednavis.vaadin.showcase.jms;
+
+public class StartMessage {
+
+  public static void main(String[] args) throws Exception {
+    final MessageSender sender = new MessageSender();
+    final MessageReceiver receiver = new MessageReceiver();
+    receiver.startListener();
+    for (int i = 1; i <= 5; i++) {
```

Comment:
We would need some UI stuff to demonstrate that JMS is working. Having main() is not acceptable.